rosa_install_access_delete_clusters/terraform/rosa-classic-creating-a-cluster-quickly-terraform.adoc

```diff
@@ -0,0 +1,34 @@
+:_content-type: ASSEMBLY
+[id="rosa-classic-creating-a-cluster-quickly-terraform"]
+= Creating a default ROSA (classic architecture) cluster using Terraform
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-classic-creating-a-cluster-quickly-terraform
+
+toc::[]
+
+Quickly create a {product-title} (ROSA) cluster quickly by using a Terraform cluster template that is configured with the default cluster options.
```

Comment:
```suggestion
Create a {product-title} (ROSA) (classic architecture) cluster quickly by using a Terraform cluster template that is configured with the default cluster options.
```
The double set of parentheses definitely contradicts both our style guide and simplistic, minimal writing. However, because "ROSA" is an acronym, we need to spell it out at first mention and put the acronym in parentheses. The two cannot be combined into a single set of parenthesis because "classic architecture" is not part of the ROSA acronym. You may want to initiate further discussion how we are going to handle situations such as these.